helm-charts/helm3/strimzi-kafka-operator/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v2
-appVersion: 0.21.1
-description: 'Strimzi: Apache Kafka running on Kubernetes'
-home: https://strimzi.io/
+appVersion: "0.1.0"
+description: "Strimzi: Apache Kafka running on Kubernetes"
+name: strimzi-kafka-operator
+version: 0.1.0
```

Comment:
Is there somewhere I can read about this apparent reversion from 0.22 to 0.1?

Reply:
I guess that looks like some bug? Either what was there before or what is there now?

Reply:
I opened #4590 to fix it. Feel free to comment there.